src/views/projects/Patch.svelte

```diff
@@ -136,7 +142,8 @@
   }));
 
   $: currentRevision =
-    patch.revisions.find(r => r.id === revision) || patch.revisions[0];
+    patch.revisions.find(r => r.id === (revision ?? latestRevision)) ||
+    patch.revisions[0];
```

Comment:
How about this:
```suggestion
    (revision && patch.revisions.find(r => r.id === revision) || patch.revisions[patch.revisions.length - 1]
```
Then you can get rid of `latestRevision` and the intention becomes much clearer.

Reply:
069d0a16ce2fc8355c363dbbb80477a365240ffb